pkg/tfbridge/config_encoding.go

```diff
@@ -98,9 +92,6 @@ func (enc *ConfigEncoding) convertStringToPropertyValue(s string, typ shim.Value
 		}
 
 		v := resource.NewPropertyValueRepl(encodedSecret, nil, replv)
-		if opts.KeepSecrets {
-			v = resource.MakeSecret(v)
```

Comment:
These are the same changes that were making me uncomfortable in https://github.com/pulumi/pulumi-terraform-bridge/pull/2195 - can we please hold until I get a chance to review this step by step. It seems to me that it's fairly brittle part of the system and it's not been adequately tested.

Reply:
Ok looks like this is a dead code, but it's concerning to lose this in refactoring. Let's maybe throw in an assert that opts.KeepSecrets is false here?